package.json

```diff
@@ -39,5 +39,9 @@
     "sharp": "0.33.0",
     "gatsby-sharp": "1.12.0"
   },
-  "packageManager": "yarn@3.2.3"
+  "packageManager": "yarn@3.2.3",
+  "dependencies": {
+    "@adobe/react-spectrum": "^3.35.0",
```

Comment:
I'm not sure about including react-spectrum as a dependency. It looks like we are just using it for a few ui components. I would like to not have react-spectrum as it would increase the bloat/size of the theme for all who consume it. Moreover, most people who do use the theme won't be using the create credentials component. 

I think moving forward we would want to remove whatever ui components we are getting from it and just create our own version of those ui components.